.github/workflows/clean-aws-resources.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+
+name: Monthly Resources Cleaner
+
+on:
+  schedule:
+    - cron: "0 0 1 * *" # Run on the first day of every month
+  workflow_dispatch:
+    inputs:
+      daystokeep:
+        description: "days to keep a resource"
+        required: false
+        default: "30"
+
+env:
+  DAYS_TO_KEEP: 30
+
+jobs:
+  clean-resources:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        resource: [ami, dedicated_host, ssm]
+        region: [ us-east-1, us-east-2, us-west-2, eu-west-1 ]
```

Comment:
Do our integration tests on GitHub create any resources in a region outside of us-west-2? What is this for?

Reply:
Ack. I initially thought the our open-source integration-test uses the same account as the internal integration test. Therefore, cleaning in those regions. However, I have changed to only us-west-2 in those commits.